src/business_logic/transaction/error.rs

```diff
@@ -39,6 +39,10 @@ pub enum TransactionError {
 
     #[error("Calling other contracts during validate execution is forbidden")]
     UnauthorizedActionOnValidate,
+    #[error("The entry_point_selector must be 617075754465154585683856897856256838130216341506379215893724690153393808813, found {0:?}")]
+    UnauthorizedEntryPointForInvoke(Felt),
+    #[error("Error ExecutionEntryPoint")]
+    ExecutionEntryPointError,
```

Comment:
Are you using these errors?

Reply:
Not me. I think they were there from a previous merge to main. Removed.